bin/check_ldap.rb

```diff
@@ -0,0 +1,21 @@
+# This is a rails runner that will print the status of the database
+# Possible outcomes:
+#
+#   * `LDAP_READY`: connection successfull
+#   * `LDAP_FAIL`: cannot authenticate within ldap server
+#   * `LDAP_DOWN`: cannot connect to the ldap server
+
+def ldap_ready?
+  ldap_config = Velum::LDAP.ldap_config
+  ldap = Net::LDAP.new(ldap_config)
+
+  if ldap.bind
+    puts "LDAP_READY"
+  else
+    puts "LDAP_FAIL"
+  end
+rescue Errno::ECONNREFUSED
+  puts "LDAP_DOWN"
+end
+
+ldap_ready?
```

Comment:
Is it really needed to have it as a method?

Reply:
The method need to be called somehow unless we don't use a method declaration,

Reply:
What I meant is that you don't need to implement it as a method and then call it: you can simply have the code as-is :)

Reply:
I know, I just followed the same pattern as `check_db.rb` file.